src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -112,7 +85,7 @@ private enum BatchWriterState
             BatchStarted,
 
             /// <summary>WriteStartChangeSet has been called.</summary>
-            ChangeSetStarted,
+            ChangesetStarted,

```

Comment:
>ChangesetStarted [](start = 12, length = 16)

why change the casing here?

Reply:
One of the build utilities (stylecop or CA?) gave error about the original name of not being discrete, and suggest to change to the new name.

Reply:
I believe Code style complained about "ChangeSetStarted" is an existing distinct word and suggested the change.

---
In reply to: [140611567](https://github.com/OData/odata.net/pull/960#discussion_r140611567) [](ancestors = 140611567)